README.md

```diff
@@ -33,6 +33,20 @@ With stylelint, it's easy to start linting your CSS:
 4. Decide whether to use the [CLI](/docs/user-guide/cli.md), [Node API](/docs/user-guide/node-api.md), or [PostCSS plugin](/docs/user-guide/postcss-plugin.md). Be sure to [specify the syntax](/docs/user-guide/css-processors.md#parsing-scss) if you're using SCSS.
 5. Lint!
 
+## Help out
+
+There is always a lot of work to do, and already well over 100 rules to maintain. So please help out in any way that you can!
+
+You can help stylelint grow in a number of ways:
```

Comment:
Can we drop "stylelint grow" and just go with: 

"You can help in a number of ways:"

As growth feels like a by-product of making a good tool, rather than a goal in itself.